cmd/osbuild-worker/jobimpl-file-resolve.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/osbuild/osbuild-composer/internal/remotefile"
+	"github.com/osbuild/osbuild-composer/internal/worker"
+	"github.com/osbuild/osbuild-composer/internal/worker/clienterrors"
+)
+
+type FileResolveJobImpl struct{}
+
+func (impl *FileResolveJobImpl) Run(job worker.Job) error {
+	logWithId := logrus.WithField("jobId", job.Id())
+	var args worker.FileResolveJob
+	err := job.Args(&args)
+	if err != nil {
+		return err
+	}
+
+	result := worker.FileResolveJobResult{
+		Contents: make([][]byte, len(args.URLs)),
+	}
+
+	defer func() {
+		logWithId := logrus.WithField("jobId", job.Id().String())
+		if result.JobError != nil {
```

Comment:
In other job types, we have the `success` field, which we set by default to `false` at the beginning. This job type does not have that. I don't see any case when the function would return without setting an error, but I wonder whether we should set it in this deferred function if the `result.Contents` is empty? It probably does not make make sense to submit a job that resolves zero files, thus returning empty results seems like an error...